### PR TITLE
Enable admin mode on qa

### PR DESCRIPTION
**Pull request by mafonso**
### Context

This builds on top of the Admin Conditional deployment to enable it on QA

### CODE REVIEW

File: azure-pipelines-deploy-template.yml [hunk 1]
```diff
@@ -68,3 +68,34 @@ jobs:
                 SpecifySlotOrASE: true
                 resourceGroupName: '${{parameters.resourceGroupName}}'
                 Slot: staging
+
+            - task: AzureAppServiceManage@0
+              displayName: 'Start Azure Admin Service: ${{parameters.environment}}'
+              condition: ne(variables['appServiceName'], 'disabled')
+              inputs:
+                azureSubscription: '${{parameters.subscriptionServiceConnection}}'
+                Action: 'Start Azure App Service'
```
Comment (cwrw) [inline azure-pipelines-deploy-template.yml]: can we change the action to admin service as well, here and below?

Comment (mafonso) [reply azure-pipelines-deploy-template.yml]: The action is a reserved "keyword" for the task.  
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-app-service-manage.

File: azure-pipelines-deploy-template.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 parameters:
-  debug: false
+  debug: true
```
Comment (cwrw) [inline azure-pipelines-deploy-template.yml]: do we need to keep this as true?

Comment (mafonso) [reply azure-pipelines-deploy-template.yml]: No, I can remove it. It does not seem to work anyway. That should give you more details on the job output, which is handy to have enabled.
I'll disable it again and revisit this later.

File: azure-pipelines-deploy-template.yml [hunk 3]
```diff
@@ -36,11 +36,39 @@ jobs:
...
+                      } elseif ($type -eq "string") {
+                          Write-Host "##vso[task.setvariable variable=$keyname]$value"
+                          Write-Host "Added Azure DevOps variable '$keyname' ('$type') with value '$value'"
+                      } else {
+                          Throw "Type '$type' is not supported for '$keyname'"
+                      }
+                  }
+                }
+                Convert-ArmOutputToPsObject -ArmOutputString '$(ArmOutputs)'
+              displayName: "Parsing outputs from ARM Deployment into pipeline variables"
```
Comment (tavy88) [inline azure-pipelines-deploy-template.yml]: Interesting script - take a bit of time to digest.

**Pull request merged**